lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
             padding: EdgeInsets.only(bottom: 32),
           ),
           Expanded(
-            child: Stack(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
               children: <Widget>[
-                Center(
-                  child: Container(
-                    height: 280.0,
-                    width: 280.0,
-                    child: CardMaturityDonutChart(
-                      seriesList: <charts.Series<MatureCount, int>>[
-                        charts.Series(
-                          id: 'CardMaturity',
-                          domainFn: (MatureCount matures, _) =>
-                              matures.maturity,
-                          measureFn: (MatureCount matures, _) =>
-                              matures.maturity,
-                          colorFn: (_, i) => charts.Color(
-                                r: SarakaColors.lightRed.red,
-                                g: SarakaColors.lightRed.green,
-                                b: SarakaColors.lightRed.blue,
-                                a: 180,
-                              ),
-                          data: [
-                            new MatureCount(
-                                "More than 80%",
-                                cardsMaturity
-                                    .where((iter) => iter.maturity * 100 >= 80)
-                                    .toList()
-                                    .length),
-                            new MatureCount(
-                                "50%",
-                                cardsMaturity
-                                    .where((iter) =>
-                                        iter.maturity * 100 >= 50 &&
-                                        iter.maturity * 100 < 80)
-                                    .toList()
-                                    .length),
-                            new MatureCount(
-                                "30%",
-                                cardsMaturity
-                                    .where((iter) =>
-                                        iter.maturity * 100 >= 30 &&
-                                        iter.maturity * 100 < 50)
-                                    .toList()
-                                    .length),
-                            new MatureCount(
-                                "Less than 30%",
+                Stack(
+                  alignment: Alignment.center,
+                  children: <Widget>[
+                    Container(
+                      height: 280.0,
+                      width: 280.0,
```

Comment:
We need to change it to be responsible (use MediaQuery.size) in another pull request